WordPress/src/androidTest/java/org/wordpress/android/ui/screenshots/WPLocaleTestRule.java

```diff
@@ -45,6 +46,7 @@ public void evaluate() throws Throwable {
 
     private static void changeLocale(String localeCode) {
         LocaleManager.setNewLocale(InstrumentationRegistry.getTargetContext(), localeCode);
+        WordPress.updateContextLocale();
```

Comment:
This is the change to ensure the Application's context when locale is changed. Related to the change I made in https://github.com/wordpress-mobile/WordPress-Android/pull/8116. I didn't add this at the time because `develop` needed to be merged in.

Reply:
Given that this is already approved by @AmandaRiu, I'm good with this, even though I don't understand it 😅